resources.whatwg.org/file-issue.js

```diff
@@ -10,43 +10,44 @@
 
 (function () {
   'use strict';
-  var thisScript = document.currentScript;
 
-  var originalFilingUrl = getOriginalFilingUrl();
-  var titlePrefix = '';
-  var queryParamIndex = originalFilingUrl.indexOf('?title=');
+  let originalFilingUrl = getOriginalFilingUrl(),
+      titlePrefix = '';
```

Comment:
Style-wise I am not comfortable maintaining scripts that use comma-separated variable declarations; it's too easy to miss the comma and then end up declaring a global variable.